### ISSUE TITLE
"Modal Checkout" not supported in Chrome 63

### ISSUE THREAD

**Issue by sam-oxipay** [oxipay/oxipay-woocommerce#99]
*Steps to replicate*
1 - Setup a WooCommerce store with the Oxipay installed and plugin enabled.
2 - Enable 'Modal Checkout'.
3 - Perform a transaction with Oxipay as your payment method.

*Expected result*
Modal Checkout would be enabled as per the configuration within the Oxipay plugin.

*Actual result*
User is presented with a pop-up message indicating that browsing in Private Mode is not supported as per screenshot below:

![chrome_63_privatebrowsing](https://user-images.githubusercontent.com/22509613/33415537-2a50ee18-d5e7-11e7-9c10-9902045ae276.png)

**Comment by am-oxipay** [comment]
I don't think this is a major issue as modal is not enabled by default, additionally block third party cookies is not enabled by default.

I also don't really see how this can work in modal form without 3rd party cookies enabled so I don't really have a suggested work around if sellers want to support this.
